operator/test/test.sh

```diff
@@ -288,6 +288,13 @@ test_security() {
 }
 
 test_results() {
+  # Check logs for OCP bug https://issues.redhat.com/browse/OCPBUGS-5916
+  pattern="http2: server: error reading preface from client +.+ read: connection reset by peer"
+  if kubectl logs deployment/tekton-results-api -c "api" -n "$NAMESPACE" 2>&1 | grep -ciqE "$pattern"; then
```

Comment:
I assume this `grep` is right, but I have no idea :-) 

You got a quick sample to confirm @sayan-biswas ?

Or, was there a reason checking for `http2: server: error reading preface from client` was not sufficient ?

Reply:
Ya, its right, tested it. I used as a regular expression.
But I'll update and just put `http2: server: error reading preface from client` instead, to catch broader errors. I guess the rest of the messages are unwrapped errors from other methods down the stack.
I added few messages also, so I am pushing the PR again.

Reply:
done